### PR TITLE
Adds Tests

### PR DESCRIPTION
**Pull request by PadgeM**

### CODE REVIEW

File: src/__tests__/Footer.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  //       <Footer />
+  //     </BrowserRouter>
+  //   )
+  //   userEvent.click(screen.getByText("Meet the Cats"))
+  //   expect(screen.getByText("Meet the Cats")).toBeInTheDocument()
+  //   userEvent.click(screen.getByText("Add a New Cat"))
+  //   expect(screen.getByText("Add a New Cat")).toBeInTheDocument()
+  //   userEvent.click(screen.getByText("Adopt a Cat!"))
+  //   expect(screen.getByText("Adopt a Cat!")).toBeInTheDocument()
+  // })
```
Comment (elysemontano) [inline src/__tests__/Footer.test.js:35]: Right now this file is not testing anything.  Make sure to comment these back in and make them pass.

File: src/__tests__/Home.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  });
+
+  // Reusable setup function
+  function setup() {
+    const utils = render(<Home />);
+    return {
+      ...utils,
+      // Add any additional utility functions or elements needed for the tests
+    };
+  }
```
Comment (elysemontano) [inline src/__tests__/Home.test.js:29]: Not seeing anything being asserted in these tests.  Make sure you have some expect statements that are asserting and remove commented-out code.

**Review by elysemontano — CHANGES_REQUESTED** [review]
It looks like there is a lot of commented out code in your tests and many are not asserting content creating a false pass.

**Comment by PadgeM** [comment]
Sad days

**Pull request closed without merging**